### DISCUSSION TITLE
Next.js app error when deploying to Vercel

### DISCUSSION THREAD

**Discussion by hilmanski** [vercel/community#486] [Help] [answered]
Hi I tried npm run build or yarn run build
everything works at my local machine

But when I tried to deploy it to Vercel, this error shows up
```
ModuleNotFoundError: Module not found: Error: Can't resolve '../components/layout' in '/vercel/path0/pages'
```

I've make sure that layout using capital letter 'Layout' , 
I even try to delete the layout first then comment,
Then create new Layout file again to re-commit.
Still same error shows up

**Comment by hilmanski** [comment, marked as answer]
So, the uppercase should happen when importing
import Layout from './components/Layout'

maybe because of linux case sensitive  rule

**Reply by devSahinur** [reply]
Brother, can you solve this problem?
